OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -269,6 +269,12 @@ public interface INotifyDeployTriggered
 		void Undeploy(Actor self, bool skipMakeAnim);
 	}
 
+	public interface IResourceAccumulatorInfo : ITraitInfoInterface { }
+	public interface IResourceAccumulator
+	{
+		int AcceptResources(string resourceType, int count = 1);
```

Comment:
```suggestion
		int AcceptResources(string resourceType, int count);
```
Your default implementation doesn't use the default parameter either.

Reply:
This should also be removed from `IAcceptResources`. `Harvester` should look for `ResourceAccumulator` instead